README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+
+# hw01
+
+My name is **Bingxin** (Janssie) Zhu. In this course, I  want to 
+
+* Learn how to code using R
+* Conduct data anaglsis using R 
+
+#### My research interests 
+
+I am  interested in the space structure in rural China. Here is a [link](https://www.landesa.org) of current land policy report in China. 
```

Comment:
This is also a really interesting research interest! Your work sounds fascinating.